src/main/java/ru/yandex/qatools/allure/jenkins/utils/FilePathUtils.java

```diff
@@ -38,4 +44,34 @@ public static void deleteRecursive(FilePath filePath, PrintStream logger) {
             logger.println(String.format("Can't delete directory [%s]", filePath));
         }
     }
+
+    public static InputStream getFileInputStream(final FilePath path, final TaskListener listener) {
+        try {
+            return path.read();
+        } catch (IOException | InterruptedException e) {
+            listener.getLogger().println("Unable to read file: " + e);
+        }
+        return null;
+    }
+
+    public static byte[] computeSha1Sum(final InputStream stream, final TaskListener listener) {
```

Comment:
this method should get `FilePath` as parameter, and return String. To get string representation of checksum use the following function:

```java
new BigInteger(1, bytes).toString(16) 
```